tensorrt_llm/serve/chat_utils.py

```
@@ -199,7 +199,9 @@ def parse_chat_messages_coroutines(
 
 
-def check_multiple_response(n: int, backend: Optional[str]):
-    if n > 1 and backend == "pytorch":
-        raise ValueError(
-            "Multiple response is not supported in PyTorch workflow")
+def make_tool_call_id(id_type: str = "random", func_name=None, idx=None):
+    if id_type == "kimi_k2":
+        return f"functions.{func_name}:{idx}"
+    else:
+        # by default return random
+        return f"chatcmpl-tool-{uuid.uuid4().hex}"
```

Ruff (0.14.5)

Undefined name `uuid`

(F821)